ironfish-cli/src/commands/deposit.ts

```diff
@@ -108,15 +108,16 @@ export default class Bank extends IronfishCommand {
     const displayAmount = displayIronAmountWithCurrency(IRON_TO_SEND, true)
     const displayFee = displayIronAmountWithCurrency(feeInIron, true)
     const displayNewBalance = displayIronAmountWithCurrency(newBalance, true)
-    if (!flags.confirm) {
-      this.log(`
-You are about to send ${displayAmount} plus a transaction fee of ${displayFee} to the Iron Fish deposit account.
+
+    this.log(`
+Your balance is ${confirmedBalance} and you are about to send ${displayAmount} plus a transaction fee of ${displayFee} to the Iron Fish deposit account.
```

Comment:
One other suggestion! It would be nice if `confirmedBalance` was formatted the same way that the other values are ☝️

Reply:
@hughy done. Corrected the format too!

Reply:
@hamza-siddiq , I meant that the other values use `displayIronAmountWithCurrency`

That way a confirmed balance of $IRON 1 will be displayed as `$IRON 1.00000000 ($ORE 100,000,000)` instead of `100000000`

Reply:
Aah, I see. I'll work on it

Reply:
@hughy done. Thanks for the suggestion. It should be good now